ldap-connector-demo/processes/demo.mod

```diff
@@ -0,0 +1,70 @@
+[Ivy]

```

Comment:
I think at the end we need an HTML Dialog as demo ;)

Where you can enter a query string..